src/test/java/Domashka17.java

```diff
@@ -0,0 +1,42 @@
+import com.fasterxml.jackson.databind.ser.Serializers;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Domashka17 extends BaseTest {
+
+
+    @Test
+    public static void addSongToThePlaylist() {
+        BaseTest B = new BaseTest();
+        B.setupBrowser();
+        B.navigation("https://bbb.testpro.io/");
+        B.enterEmail("valentineguts@gmail.com");
+        B.enterPassword("te$t$tudent");
+        B.submit();
+        B.search("hozlinaCC0");
+       // B.sleep();
+        B.clickViewALl();
+        B.clickFirstSong();
+        B.AddTo();
+        B.addToPlaylist();
+        Assert.assertTrue(B.verifyNotification().contains("Added 1 song into"));
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._